lib/slotty.rb

```diff
@@ -34,10 +42,14 @@ def get_slots(for_range:, slot_length_mins:, interval_mins:, as: :full, exclude_
 
         slots << potential_slot.send(as)
 
-        potential_slot = Slot.new(range: (potential_slot.begin + interval)..(potential_slot.begin + interval + slot_length))
+        potential_slot = slot_for_range(range: (potential_slot.begin + interval)..(potential_slot.begin + interval + slot_length), meta:)
       end
 
       slots
     end
+
+    def slot_for_range(range:, meta:)
+      Slot.new(range:, meta:)
```

Comment:
Also, this won't work.
```suggestion
      Slot.new(range: range, meta: meta)
```

Reply:
Actually I'm just following the default setting for the Rubocop convention `Style/HashSyntax` here (EnforcedShorthandSyntax: always). 

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashSyntax

```ruby
# bad
{foo: foo, bar: bar}

# good
{foo:, bar:}
```

But I just learnt that it was added in Ruby 3.1, so yes it needs to be removed. https://dev.to/baweaver/ruby-3-1-shorthand-hash-syntax-first-impressions-19op.